docs/shared/npm-tutorial/package-based.md

```diff
@@ -153,59 +160,77 @@ Remove any existing `dist` folder and run:
 npx nx build is-odd
 ```
 
-It will automatically first run `npx nx build is-even` and then the build for `is-odd`. Note that if `is-even` has been built before, it would just be restored out of the cache.
+It will automatically first run `build` for the `is-even` package, and then the `build` for `is-odd`. Note that if `is-even` has been built before, it would just be restored out of the cache.
 
 ## Cache Build Results
 
-To build the `is-even` package run:
-
-```shell
-npx nx build is-even
-```
-
-Run the same command a second time and you'll see the build cache is being used:
+Run the command:
 
 ```{% command="npx nx build is-even" %}
-> nx run is-even:build
+> nx run is-even:build  [existing outputs match the cache, left as is]
+
 
-> @package-based/is-even@0.0.0 build
+> is-even@0.0.0 build
 > tsc index.ts --outDir dist
 
-—————————————————————————————————————————————————————————————————
 
-> NX Successfully ran target build for project is-even (1s)
+ ——————————————————————————————————————————————————————————————————————————————————————————
+
+ >  NX   Successfully ran target build for project is-even (33ms)
+
+   Nx read the output from the cache instead of running the command for 1 out of 1 tasks.
 
 ```
 
+Note that the cache for the `build` script was already populated when we ran it previously in this tutorial.
+
 ## Running Multiple Tasks
 
 To run the `build` target for all the packages in the workspace, use:
 
-```shell
-npx nx run-many --target=build
-```
-
-What you would get is the following:
-
 ```{% command="npx nx run-many --target=build" %}
     ✔  nx run is-even:build  [existing outputs match the cache, left as is]
-    ✔  nx run is-odd:build (906ms)
+    ✔  nx run is-odd:build  [existing outputs match the cache, left as is]
```

Comment:
cache for `is-odd` is already warmed as we had the users build `is-odd` in the prior step.